tests/service/subscriptions/mqtt/test_subscription_service.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+from agrirouter.messaging.parameters.service import SubscriptionParameters
+from agrirouter.generated.messaging.request.payload.endpoint.subscription_pb2 import Subscription
+from agrirouter.messaging.services.messaging import SubscriptionService
+from agrirouter.messaging.services.commons import MqttMessagingService
+from agrirouter.messaging.decode import decode_response
+from agrirouter.messaging.messages import OutboxMessage
+from tests.data.onboard_response_integration_service import OnboardResponseIntegrationService
+from agrirouter.utils.uuid_util import new_uuid
+from agrirouter.messaging.services.sequence_number_service import SequenceNumberService
+from tests.data.identifier import Identifier
+from agrirouter.messaging.enums import CapabilityType
+from tests.sleeper import Sleeper
+
+
+class TestMqttSubscriptionService:
+
+    def test_when_sending_subscriptions_for_pem_recipient_then_the_sender_should_accept_them(self):
+        """
+        Load the existing onboard response and send the subscriptions via mqtt
+        """
+        _onboard_response = TestMqttSubscriptionService.load_onboard_response(
+            Identifier.MQTT_RECIPIENT_PEM[Identifier.PATH])
+        TestMqttSubscriptionService._send_subscriptions_via_mqtt(onboard_response=_onboard_response,
+                                                                 mqtt_message_callback=TestMqttSubscriptionService._on_message_callback)
+
+    @staticmethod
+    def load_onboard_response(path):
```

Comment:
This is a wrapper of a single line call, more or less useless. Use the function directly.